### PR TITLE
Adding new guide for the 11ty framework

### PR DESCRIPTION
**Pull request by jkup**
I copy / pasted from https://developers.cloudflare.com/pages/framework-guides/deploy-a-gatsby-site and made some changes. I also deployed a demo of it to https://11ty-pages.pages.dev/ but wasn't sure if it would be appropriate to link to that or not?

cc @rita3ko @deadlypants1973

### CODE REVIEW

File: products/pages/src/content/framework-guides/deploy-an-eleventy-site.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+pcx-content-type: how-to
+---
+
+# Deploy an Eleventy site
+
+Eleventy is a simpler static site generator. In this guide, you will create a new Eleventy site and deploy it using Cloudflare Pages. You will be using the `eleventy` CLI to create a new Eleventy site.
```
Comment (rita3ko) [inline products/pages/src/content/framework-guides/deploy-an-eleventy-site.md]: s/ simpler / simple

Comment (rita3ko) [reply products/pages/src/content/framework-guides/deploy-an-eleventy-site.md]: (we seem to be inconsistent about doing this across the different guides, but would be good to link to eleventy's site)

**Review by lukeed — APPROVED** [review]

File: products/pages/src/content/framework-guides/deploy-an-eleventy-site.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+Install the `eleventy` CLI by running the following command in your terminal:
+
+```sh
+$ npm install -g @11ty/eleventy
+```
+
+## Creating a new project
+
+There are a lot of [starter projects](https://www.11ty.dev/docs/starter/) available on the Eleventy website. As an example, use the `eleventy-base-blog` project by running the following commands in your terminal.
```
Comment (rita3ko) [inline products/pages/src/content/framework-guides/deploy-an-eleventy-site.md]:
```suggestion
There are a lot of [starter projects](https://www.11ty.dev/docs/starter/) available on the Eleventy website. As an example, use the `eleventy-base-blog` project by running the following commands in your terminal:
```

**Review by rita3ko — CHANGES_REQUESTED** [review]
Looks great! Left a couple minor edits / suggestions.

**Comment by jkup** [comment]
@rita3ko updated!

**Review by rita3ko — APPROVED** [review]
woot!

**Pull request merged**